jck/README.md

```diff
@@ -10,13 +23,28 @@
 1. Put unarchived jck test materials (jck8b or jck9) into an empty folder, for example:
 * `/jck/jck8b/` and `/jck/jck9`
 
-2. Export `JCK_ROOT=/jck` as an environment variable or pass it in makefile when run make command
+2. Export `JCK_ROOT=/jck` as an environment variable or pass it in makefile when run make commands
 
-3. Export `JAVA_HOME=<your_JDK_root>` as an environment variable
+3. Export `JCK_VERSION=<your_jck_version>` as an environment variable or pass it in makefile when run make commands. For example `export JCK_VERSION=jck8b` 
 
-4. The other steps will stay the same as instructed in `openj9/test/README.md`
+4. Export `JAVA_HOME=<your_JDK_root>` as an environment variable
+
+5. The other steps will stay the same as instructed in `openjdk-tests/README.md`
 
 
 This test directory contains:
   * build.xml file - that clones AdoptOpenJDK/stf repo to pick up a test framework
   * playlist.xml - to allow easy inclusion of JCK tests into automated builds
+
+
+# How-to Run customized JCK test targets
+
+There are three custom JCK test targets `jck-runtime-custom`, `jck-runtime-custom` and `jck-runtime-custom`. With these three test targets, you can run custom JCK subsets.
```

Comment:
Is this a cut'n'paste error, the 3 targets have identical names?  do you mean to include jck-devtools-custom and jck-compiler-custom in that list.

Reply:
ah, thanks Shelley, this is a cut'n error. updated with newer changeset.